### PR TITLE
Cleanup Storage

### PR DESCRIPTION
**Pull request by bkcsoft**
- Happy-path to the left
- StdoutPipe all the things! :fist_raised:

### CODE REVIEW

File: cmd/storage/cli.go [hunk 1]
```diff
@@ -19,17 +19,17 @@ func treeAction(c *cli.Context) error {
 	}
 	owner, name, rev, path := args[0], args[1], args[2], args[3]
 
-	s, err := storage.NewLocalStorage(c.GlobalString(cmd.FlagRoot))
+	stor, err := storage.NewLocalStorage(c.GlobalString(cmd.FlagRoot))
```
Comment (metalmatze) [inline cmd/storage/cli.go]: I don't really see a reason to rename this.
```suggestion
	s, err := storage.NewLocalStorage(c.GlobalString(cmd.FlagRoot))
```

File: cmd/storage/cli.go [hunk 2]
```diff
@@ -19,17 +19,17 @@ func treeAction(c *cli.Context) error {
...
 	if err != nil {
 		return err
 	}
 
-	r, err := s.GetRepository(context.Background(), owner, name)
+	repo, err := stor.GetRepository(context.Background(), owner, name)
```
Comment (metalmatze) [inline cmd/storage/cli.go]: I don't really see a reason to rename this.
```suggestion
	r, err := stor.GetRepository(context.Background(), owner, name)
```

Note: I would prefer spelling it out as `repository` over `repo`. :relaxed:

Comment (bkcsoft) [reply cmd/storage/cli.go]: `repository` overlaps with `pkg/repository` hence why I opted for `repo` instead.

**Comment by metalmatze** [comment]
Needs to be rebased now...

**Comment by bkcsoft** [comment]
Rebased :slightly_smiling_face: 

/cc @metalmatze

**Pull request merged**